package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "atip",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview",
+    "fmt": "prettier --write *.html *.js css/*"
```

Comment:
This is using globally installed prettier right now, it should really be a dev dependency. Will clean later